src/pcre2.h.in

```diff
@@ -473,7 +473,7 @@ released, the numbers must not be changed. */
 #define PCRE2_CONFIG_BSR                     0
 #define PCRE2_CONFIG_JIT                     1
 #define PCRE2_CONFIG_JITTARGET               2
-#define PCRE2_CONFIG_LINKSIZE                3
+#define PCRE2_CONFIG_LINKSIZE                3  /* Deprecated */
```

Comment:
I don't think there's a reason to deprecate the old item. It still works, even if it's not as useful as your new EFFECTIVE_LINKSIZE.